CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### 🚀 Added
 
 ### 🔧 Changed
+- Minimizing Rust Binary Size [#129](https://github.com/mgrachev/dotenv-linter/pull/132)
```

Comment:
Please replace the PR number from `#129` to `#132` and add information about you at the end of this line. For example:
```
- Refactor `Github Actions` and reduce `Docker Image size` [#90](https://github.com/mgrachev/dotenv-linter/pull/90) ([@Macbet](https://github.com/Macbet))
```